### PR TITLE
Setup Travis CI

### LINKED ISSUE #8: Setup CI
Setup Travis CI  to build and run tests when a PR is submitted.

### PR DESCRIPTION
**Pull request by caroltr**
**What this PR does**:
This PR adds basic setup to integrate Travis CI on the project.
At each Pull Request, it will build and run unit and instrumented tests.
Configurated to run instrumented tests in an Android 29 emulator and armeabi-v7a ABI.

**Which issue(s) this PR fixes**:

Fixes #8

### CODE REVIEW

File: .travis.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+language: android
+jdk: oraclejdk8
+
+env:
+  global:
+    - ANDROID_TARGET=android-22
```
Comment (stavares843) [inline .travis.yaml]: should this be higher then 22?

File: local.properties [hunk 2]
```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Tue May 12 18:23:39 BRT 2020
+sdk.dir=/Users/caroltr/Library/Android/sdk
```
Comment (stavares843) [inline local.properties]: do we need this file?

Comment (caroltr) [reply local.properties]: No we don't.
Gonna check gitignore file. Thanks!

File: .travis.yaml (hunk 1 above)
Comment (caroltr) [reply .travis.yaml]: Gonna increase to the max supported: 29.

**Pull request merged**